DQM/Integration/python/clients/beamspotdip_dqm_sourceclient-live_cfg.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import print_function
+import FWCore.ParameterSet.Config as cms
+
+#
+process = cms.Process("BeamSpotDipServer")
+process.load("DQMServices.Core.DQM_cfg")
+
+# input
+# for live online DQM in P5
+process.load("DQM.Integration.config.inputsource_cfi")
+# for testing in lxplus
+# process.load("DQM.Integration.config.fileinputsource_cfi")
+
+# beamspot from database
+process.load("CondCore.CondDB.CondDB_cfi")
+
+process.BeamSpotDBSource = cms.ESSource("PoolDBESSource",
+  process.CondDB,
+  toGet = cms.VPSet(
+    cms.PSet(
+      record = cms.string('BeamSpotOnlineLegacyObjectsRcd'),
+      tag = cms.string("BeamSpotOnlineTestLegacy"),
+      refreshTime = cms.uint64(1)
+    ),
+    cms.PSet(
+      record = cms.string('BeamSpotOnlineHLTObjectsRcd'),
+      tag = cms.string("BeamSpotOnlineTestHLT"),
+      refreshTime = cms.uint64(1)
+    )
+  )
+)
+
+process.BeamSpotESProducer = cms.ESProducer("OnlineBeamSpotESProducer")
+process.BeamSpotDBSource.connect = cms.string('frontier://FrontierProd/CMS_CONDITIONS')
```

Comment:
In assignment, explicit types on the right hand side should be avoided
http://cms-sw.github.io/cms_coding_rules.html#6--packaging-rules-1

Reply:
Will be fixed.